tools/integration/test/lib/harvesterTest.js

```diff
@@ -1,14 +1,14 @@
 // (c) Copyright 2024, SAP SE and ClearlyDefined contributors. Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
-const { strictEqual, ok } = require('assert')
+const { strictEqual, ok, deepStrictEqual } = require('assert')
 const { callFetch } = require('../../lib/fetch')
 const Poller = require('../../lib/poller')
 const Harvester = require('../../lib/harvester')
-const { devApiBaseUrl } = require('../testConfig')
+const { devApiBaseUrl, harvestToolVersions } = require('../testConfig')
 const sinon = require('sinon')
 
-describe('Integration test against dev deployment', function () {
+describe.skip('Integration test against dev deployment', function () {
```

Comment:
I think it's fine to skip these now to unblock the tests but maybe we want to either take them out completely or move to a separate file that we can run manually.

Reply:
Good point.  I will remove the skipped tests.